### PR TITLE
adds the Scottish Winter cocktail, designed by me

### PR DESCRIPTION
**Pull request by MilesMickelson**
Designed and served this drink for the speakeasy bar I used to work at. Great drink for cold October nights sure to warm you up and entice the senses.

### CODE REVIEW

**Comment by MilesMickelson** [comment]
Hope someone takes the time to make this cocktail and enjoy it.

File: src/recipes/scottish-winter.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+"image": "Coming Soon",
+"source": "Designed by Miles Mickelson",
+"keywords": [
+	"Old-fashion",
+  "Scotch",
+  "Whiskey",
+  "Peat",
+  "Smoke",
+  "Dry",
+  "Citrus"
```
Comment (alfg) [inline src/recipes/scottish-winter.json]: Can we lowercase these?

Comment (MilesMickelson) [reply src/recipes/scottish-winter.json]: Lower cased and formatted with prettier, so I think that its refactored into desired format. Also I just recently started using zshell and installed packages listed in project, pretty sure all are in desired condition I apologize if not.

**Comment by MilesMickelson** [comment]
Change requests completed

**Comment by alfg** [comment]
Closing. Feel free to re-open with requested fixes.

**Pull request closed without merging**